include/aspect/traction_boundary_conditions/lithostatic_pressure.h

```diff
@@ -0,0 +1,122 @@
+/*
+  Copyright (C) 2011, 2012 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect__traction_boundary_conditions_lithospheric_pressure_h
+#define __aspect__traction_boundary_conditions_lithospheric_pressure_h
+
+#include <aspect/traction_boundary_conditions/interface.h>
+#include <aspect/simulator_access.h>
+
+
+namespace aspect
+{
+  namespace TractionBoundaryConditions
+  {
+    using namespace dealii;
+
+    /**
+     * A class that implements traction boundary conditions based on a
+     * functional description provided in the input file.
+     *
+     * @ingroup TractionBoundaryConditionsModels
+     */
+    template <int dim>
+    class LP : public Interface<dim>, public SimulatorAccess<dim>
```

Comment:
And maybe give that class a longer name (like LithostaticPressure) so that it's easier to understand what it does.